RealifeTech-CoreSDK/GraphQL/GraphQLFiles/payment_getMyPaymentIntent.graphql

```diff
@@ -0,0 +1,30 @@
+query getMyPaymentIntent($id: ID!) {
```

Comment:
When do we use this query?

Reply:
I don't think we use this for current spec. although it's on API document. However, I'll remove this to avoid confusion. Thank you!